Podcast/Assets.xcassets/add_icon.imageset/Contents.json

```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "icon.png",
+      "scale" : "1x"
+    },
+    {
+      "idiom" : "universal",
+      "filename" : "icon@2x.png",
+      "scale" : "2x"
+    },
+    {
+      "idiom" : "universal",
+      "filename" : "icon@3x.png",
+      "scale" : "3x"
+    }
+  ],
+  "info" : {
+    "version" : 1,
+    "author" : "xcode"
```

Comment:
can you export this as a pdf instead? it'll be fewer files

Reply:
I only had jpg and png as options on zeplin.. should I check with Eileen?

Reply:
there should be an option for "vector asset" on zeplin but if not it's fine!